apps/platforms/win/explorer/explorer.py

```diff
@@ -35,21 +35,10 @@
 
 if app.platform == "windows":
     is_windows = True
-    import ctypes
-
-    GetUserNameEx = ctypes.windll.secur32.GetUserNameExW
-    NameDisplay = 3
-
-    size = ctypes.pointer(ctypes.c_ulong(0))
-    GetUserNameEx(NameDisplay, None, size)
-
-    nameBuffer = ctypes.create_unicode_buffer(size.contents.value)
```

Comment:
It looks like this is actually used in this file, so will need to leave it in.

Reply:
No problem, you're helping a lot. I restored this. Let me know if there's anything else I need to change

Reply:
I think you'll need to keep the GetUserNameEx line as well since it is probably what populates nameBuffer.

Reply:
Oh, man, sorry. I'll try to be more thorough next time.

Reply:
No worries, we all miss stuff sometimes. That's one thing that makes code review useful :).